src/Adapters/Auth/xiaomi.js

```diff
@@ -0,0 +1,65 @@
+'use strict';
+
+// Helper functions for accessing the Xiao Mi Graph API.
+var https = require('https');
+var Parse = require('parse/node').Parse;
+var crypto = require('crypto');
+
+const PRODUCTION_URL = "https://cn-api.unity.com";
+const DEBUG_URL = " https://cn-api-debug.unity.com";
```

Comment:
This url doesn't seem right, xiaomi is unity? Can you clarify this 3rd party provider and the urls?